ansible/roles/cyhy_commander/templates/commander.conf.j2

```diff
@@ -15,8 +15,8 @@ nessus-hosts = vulnscan1
 
 [production]
 database-name = cyhy
-jobs-per-nmap-host = 12
-jobs-per-nessus-host = 128
+jobs-per-nmap-host = {{ jobs_per_nmap_host }}
+jobs-per-nessus-host = {{ jobs_per_nessus_host }}
```

Comment:
I confirmed that you have already added our current Production values for these settings into our Production Terraform variables file, so thank you!  👍 👍